bases/rsptx/interactives/runestone/webwork/js/webwork.js

```diff
@@ -96,12 +96,16 @@ class WebWork extends RunestoneBase {
             if (data.rh_result.answers[k].score == 1) {
                 correctCount += 1;
             }
+            // mostly grab original_student_ans, but grab student_value for MC exercises
+            let student_ans = (['Value (parserRadioButtons)', 'Value (PopUp)' 'Value (CheckboxList)'].includes(data.rh_result.answers[k].type))
```

Comment:
I think you are missing a `,` before `'Value (Checkbox...`

Also I think the syntax is wrong.  For the conditional assignment statement operator the syntax is

`var = value_if_true : condition ? value_if_false`

Reply:
Sorry about the comma, and thanks for catching it.

I think I have the conditional operator right, or at least one way of having it right. [Here](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Operators/Conditional_operator) it's `condition ? ifTrue : ifFalse`.

I just force pushed with the comma and a little simplification.

Reply:
I think that is why I typically just use if statements.